vis-motions.c

```diff
@@ -16,9 +16,9 @@ static bool search_word(Vis *vis, Text *txt, size_t pos) {
 	char *buf = text_bytes_alloc0(txt, word.start, text_range_size(&word));
 	if (!buf)
 		return false;
-	snprintf(expr, sizeof(expr), "\\<%s\\>", buf);
+	snprintf(expr, sizeof(expr), "\\(%s\\)", buf);
 	free(buf);
-	return text_regex_compile(vis->search_pattern, expr, REG_EXTENDED) == 0;
+	return text_regex_compile(vis->search_pattern, expr, 0) == 0;
```

Comment:
If this does need to change it should probably use `REG_BASIC`:

```
$ man regex
     REG_EXTENDED  Compile modern (``extended'') REs, rather than the obsolete (``basic'') REs that are the default.
     REG_BASIC     This is a synonym for 0, provided as a counterpart to REG_EXTENDED to improve readability.
```